ruby.rb

```diff
@@ -4,42 +4,71 @@
 # 1. Use TWO different Ruby methods to return a new array with all the numbers multiplied by 3. Expected output: [3, 6, 18, 27, 9, 63]
 
 myArray = [1, 2, 6, 9, 3, 21]
+def array
+  num = 1
+  array.each do |i|
+    num *= i
+  end
+  p num
+end
 
+myArray = [1, 2, 6, 9, 3, 21]
 
+def array
+  return myArray.inject(&:*)
+end
 
 # 2. Create a method that takes in a sentence and returns a new sentence with the first letter of each word capitalized. Expected output = "Hello There, How Are You?"
 
 sentence = "hello there, how are you?"
-
+def sen(sentence)
+  sentence.upcase
+end
+  
 
 
 # 3. Create a method that takes in a string and returns a new string with all the vowels removed. Expected output = " hv n vwls"
 
 no_vowels = "I have no vowels"
 
-
-
+def rem_vowl(string)
+  string.delete("aeiou")
+end
 # 4. Look at this horrible Ruby code. Fix it to be good Ruby code.
 
-class example
+class Example
   constructor(day)
-    @day=day
+    @day = day
   end
 
   def SayHi
-    if(day === "Friday"){
+    if(day == "Friday"){
       puts "TGIF!"
     }
-    else if(day === "Monday"){
+    elsif(day == "Monday"){
       puts "Its monday again"
     }
     else{
       puts "another day"
     }
+  end
 end
 
```

Comment:
You caught quite a few errors here, but also a method name should not be uppercase, you don't need curly braces in the if/else statements, and Ruby uses an initialize method rather than a constructor